knctl/README.md

```diff
@@ -0,0 +1,50 @@
+# Knative with knctl
+
+This Task deploys (or update) a Knative service. It uses
+[`knctl`](https://github.com/cppforlife/knctl) for that, and supports
+only the `deploy` subcommand as of today.
```

Comment:
https://github.com/knative/client/blob/master/docs/cmd/kn_service_create.md
The official knative client support the service creation by image as well.

Reply:
Yes, as written in #36 the end-goal is to use `kn` instead of `knctl`. This is merely the initial proposition. I went for `knctl deploy` here, as it allowed me to re-execute this same task any number of time (and it would update the service) — I wasn't sure how to do that with `kn` at that time :angel:

Reply:
NP. I am checking whether I can make the docker image available for kn. We are good for this PR.